src/rust/catloop/mod.rs

```diff
@@ -513,7 +513,7 @@ impl CatloopLibOS {
         let runtime: Ref<CatloopRuntime> = self.runtime.borrow();
 
         // Check if the queue token came from the Catloop LibOS.
-        if let Some((ref opcode, _)) = runtime.get_catloop_qt(qt) {
+        if let Some((ref opcode, _)) = runtime.get_catloop_qd(qt) {
```

Comment:
This should be get_catloop_qt, right? Because it's a queue token that we are using to get the result?

Reply:
@iyzhang I have changed this because if you look on the signature of the function it returns `Result<(demi_opcode_t, QDesc), Fail>`. Am reading it wrong?

Reply:
Ahh, ok you are right. Sorry! Maybe we should name this get opcode? We don't seem to use the qd.